go.mod

```diff
@@ -180,6 +180,7 @@ require (
 	github.com/elastic/go-freelru v0.16.0
 	github.com/elastic/go-quark v0.3.0
 	github.com/elastic/go-sfdc v0.0.0-20241010131323-8e176480d727
+	github.com/elastic/gokrb5/v8 v8.0.0-20251031130512-6af84b230ee7
```

Comment:
This refers to the `beats-fork` branch of https://github.com/elastic/gokrb5. I would prefer it if we switch to the main branch (and squash-merge https://github.com/elastic/gokrb5/pull/1 into it), that makes it harder for the change to be accidentally reverted.

Reply:
Even though I prefer proper versions, using tags. We do it for sarama, so I'm fine with keeping it consistent